apps/console/src/extensions/i18n/models/extensions.ts

```diff
@@ -21,62 +21,6 @@ import { FormAttributes, Notification, NotificationItem } from "@wso2is/i18n";
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface Extensions {
     console: {
-        application: {
-            quickStart: {
-                technologySelectionWrapper: {
-                    subHeading: string;
-                    otherTechnology: string;
-                };
-                addUserOption: {
-                    description: string;
-                    hint: string;
-                    message: string;
-                };
-                spa: {
-                    customConfig: {
-                        heading: string;
-                        anySPATechnology: string;
-                        configurations: string;
-                        protocolConfig: string;
-                        serverEndpoints: string;
-                        clientId: string;
-                        baseUrl: string;
-                        redirectUrl: string;
-                        scope: string;
-                    };
-                    techSelection: {
-                        heading: string;
-                    };
-                };
-                twa: {
-                    common: {
-                        orAnyTechnology: string;
-                    };
-                    oidc: {
-                        customConfig: {
-                            heading: string;
-                            clientSecret: string;
-                        };
-                    };
-                    saml: {
-                        customConfig: {
-                            heading: string;
-                            issuer: string;
-                            acsUrl: string;
-                            idpEntityId: string;
-                            idpUrl: string;
-                        };
-                    };
-                };
-                m2m: {
-                    customConfig: {
-                        tokenEndpoint: string;
-                        tokenRequest: string;
-                        configurations: string;
-                    };
-                }
-            };
-        };
         applicationRoles: {
```

Comment:
@brionmario, what do you think about which namespace this should be moved to? There is currently no dedicated i18n namespace for the application roles.

Reply:
Will proceed with creating a new namespace as discussed offline.

Reply:
+1

Reply:
This is addressed with [f2167f9](https://github.com/wso2/identity-apps/pull/7320/commits/f2167f9b3757a08ee6de6bea1f47a69c0cc117e2) and 
[67b921e](https://github.com/wso2/identity-apps/pull/7320/commits/67b921eb6ecbea828ac91c20ec9e051b3b18caa1)